pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1972,6 +1972,8 @@ const (
 	BackupInvalid BackupConditionType = "Invalid"
 	// BackupPrepare means the backup prepare backup process
 	BackupPrepare BackupConditionType = "Prepare"
+	// BackupStopped means the backup was stopped, just log backup has this condition
+	BackupStopped BackupConditionType = "Stopped"
```

Comment:
When it's stopped, can it resume again? If yes, I prefer use "paused"?

Reply:
If it only use for logback, make it explicit, just like LogbackupPaused

Reply:
it can not resume again, we'd better use `Stopped`, because br has `pause` and `resume` command, we can use `paused` when we support  `pause` command in the future.